### PR TITLE
SSH: Upgrade CLI to 4.14.0

### PR DESCRIPTION
**Pull request by pvizeli**

### CODE REVIEW

**Review by frenck — APPROVED** [review]

File: ssh/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 9.2.1
+
+- Upgrade Home Assistant CLI to 4.14.0
```
Comment (ludeeus) [inline ssh/CHANGELOG.md:5]: This includes the datadisk commands which require a supervisor version that is currently not on stable

Comment (ludeeus) [reply ssh/CHANGELOG.md:5]: I can't suggest for some reason :thinking: 
Should we make the version a link?
`[4.14.0](https://github.com/home-assistant/cli/releases/tag/4.14.0)`

Comment (ludeeus) [reply ssh/CHANGELOG.md:5]: ah, it's merged already :+1:

Comment (pvizeli) [reply ssh/CHANGELOG.md:5]: right, but also not will break anything

**Pull request merged**